workspaces/scorecard/plugins/scorecard-backend-module-jira/src/constants/annotations.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Annotation key for the Jira project key.
+ * @public
+ */
+export const ANNOTATION_JIRA_PROJECT_KEY = 'jira/project-key' as const;
+
+/**
+ * Annotation key for the Jira component.
+ * @public
+ */
+export const ANNOTATION_JIRA_COMPONENT = 'jira/component' as const;
+
+/**
+ * Annotation key for the Jira label.
+ * @public
+ */
+export const ANNOTATION_JIRA_LABEL = 'jira/label' as const;
+
+/**
+ * Annotation key for the Jira team.
+ * @public
+ */
+export const ANNOTATION_JIRA_TEAM = 'jira/team' as const;
+
+/**
+ * Annotation key for the Jira custom filter.
+ * @public
+ */
+export const ANNOTATION_JIRA_CUSTOM_FILTER = 'jira/custom-filter' as const;
```

Comment:
Wdyt about

```
export enum ScorecardJiraAnnotation {
  PROJECT_KEY = 'jira/project-key',
  JIRA_COMPONENT = 'jira/component'
  // ...
}
```

I also think this annotations are so important, that I personally would move this file into the `src` folder.